src/pages/VoucherRedemption/Landing.tsx

```diff
@@ -40,7 +40,12 @@ const LandingCard = (props: Props) => {
         <br />
       </CardContainer>
       <br />
-      <Button color="#ab192e">{voucher.locations[0]}</Button>;
+      <Button color='#ab192e'>
+        {voucher.location.line1}
+        <br></br>
+        {voucher.location.line2}
+      </Button>
+      ;
```

Comment:
watch out with this `;`, I think it'll be rendered in the UI

Reply:
Good catch!